pkg/common/cns-lib/vsphere/virtualcentermanager.go

```diff
@@ -171,3 +175,21 @@ func (m *defaultVirtualCenterManager) IsExtendVolumeSupported(ctx context.Contex
 	}
 	return !is67u3Release, nil
 }
+
+// IsOnlineExtendVolumeSupported checks if online extend volume is supported or not.
+func (m *defaultVirtualCenterManager) IsOnlineExtendVolumeSupported(ctx context.Context, host string) (bool, error) {
+	log := logger.GetLogger(ctx)
+
+	// Get VC instance
+	vcenter, err := m.GetVirtualCenter(ctx, host)
+	if err != nil {
+		log.Errorf("Failed to get vCenter. Err: %v", err)
+		return false, err
+	}
+	vCenterVersion := vcenter.Client.Version
+	if vCenterVersion != cns.ReleaseVSAN67u3 && vCenterVersion != cns.ReleaseVSAN70 && vCenterVersion != cns.ReleaseVSAN70u1 {
```

Comment:
I see that we have not tested this PR against 6.7U3 and 7.0 GA. So, what is the plan to test against those vSphere releases?

Reply:
I have tested one negative and one positive scenario. How do you think 6.7U3 and 7.0 defer from 7.0U1 use case?

Reply:
The version is different.

Reply:
Will correspond with FVT team for this validation.

Reply:
Please file a task bug to the QE team.